### ISSUE TITLE
Missing file with license

### ISSUE THREAD

**Issue by abtris** [facebook/fbjs#207]
Please, provide file with correct license into repository.

Thanks

**Comment by pzrq** [comment]
+1
As a [dependency of react-dom](https://github.com/facebook/react/blob/2075139ce51836f799fee8f711fff448b2f2f43a/packages/react-dom/package.json), it would be really nice to know what LICENSE this code is released under.

**Comment by zpao** [comment]
The fbjs package is distributed with a license. I think the rest are as
well (though I'll double check that), though there is not a license file at
the root of the repo.

**Comment by mikeatlas** [comment]
Bump - a top level license would be best...

**Comment by drakmail** [comment]
+1 - I see MIT license references in header of source files:

```javascript
/**
 * Copyright (c) 2013-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

But couldn't find LICENSE file in the root directory of this source tree.

**Comment by spacehaz** [comment]
+1

**Comment by zpao** [comment]
Fixed in 4369c7dfeb1ab0de27feb3e9245f599c8edd6a5c
